crates/consensus/Cargo.toml

```diff
@@ -15,7 +15,8 @@ exclude.workspace = true
 alloy-primitives = { workspace = true, features = ["rlp"] }
 alloy-rlp.workspace = true
 alloy-eips.workspace = true
-thiserror = { workspace = true, optional = true }
+alloy-rpc-types.workspace = true
```

Comment:
this dependency should not exist. RPC types should rely on consensus, not vice versa